packages/block-library/src/file/edit.native.js

```diff
@@ -485,7 +490,13 @@ export class FileEdit extends Component {
 											/>
 											<PlainText
 												editable={ false }
-												value={ __( 'Error' ) }
+												value={
+													isEmpty(
+														uploadErrorMessage
```

Comment:
What would be the type of `uploadErrorMessage` in case if it's not the "File type not permitted " error?
I think if we know that type is `null` or `undefined`, we can skip `isEmpty` and use [??](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) operator instead: 
```js
value={uploadErrorMessage ?? __( 'Error' )}
```

Reply:
Good catch! Yes, it would be `null` or `undefined`. Resolved here 207d36f